src/commands/index.ts

```diff
@@ -65,25 +65,18 @@ function resetProjectDetectionDefaultsCommand(): void {
 
 function togglePreset(
   preset: string,
-  presetMessage: string,
+  presetKey: string,
   reverseAction: boolean = false,
   global: boolean = true): void {
 
   const value = getToggleValue(preset);
+  const action = reverseAction ? value ? 'Disabled' : 'Enabled' : value ? 'Enabled' : 'Disabled';
```

Comment:
I just want to improve the readability, I know that these are optional. But first I was confused to read it a little bit.

I would like it like this:
```js
const action = reverseAction ? (value ? 'Disabled' : 'Enabled') : (value ? 'Enabled' : 'Disabled');
```

But I'm also fine with your idea.